src/zenml/model_registries/base_model_registry.py

```diff
@@ -63,7 +63,7 @@ class ModelRegistryModelMetadata(BaseModel):
     """
 
     zenml_version: Optional[str] = None
-    zenml_pipeline_run_id: Optional[str] = None
+    zenml_run_name: Optional[str] = None
     zenml_pipeline_name: Optional[str] = None
     zenml_pipeline_uuid: Optional[str] = None
     zenml_pipeline_run_uuid: Optional[str] = None
```

Comment:
I guess this is what we call `pipeline_run_id` everywhere, does it make sense to change that as well?